lib/tapjoy/autoscaling_bootstrap.rb

```diff
@@ -111,6 +112,12 @@ def check_as_clobber(create_as_group:, clobber_as:, **unused_values)
         create_as_group && Tapjoy::AutoscalingBootstrap.group.exists && !clobber_as
       end
 
+      # See what version of the yaml we are running
+      def check_yaml_version(config)
+        fail Tapjoy::AutoscalingBootstrap::Errors::Configuration::SettingRequired.new(":version:") unless config[:version]
```

Comment:
If we do this, then we need to go back and set v1 in all the old configs, right?  I think we want it so if it doesn't have a version setting, it's assumed to be v1.